package.json

```diff
@@ -84,6 +84,7 @@
     "ejs": "^2.5.6",
     "eslint-plugin-eslint-plugin": "^0.7.1",
     "eslint-plugin-node": "^4.2.1",
+    "eslint-plugin-throws-on-load": "^1.0.0",
```

Comment:
Proxyquire supports mocking a dependency with any function. If you throw inside that function, it should work.